python_scripts/customize_octopi_config.py

```diff
@@ -14,6 +14,14 @@ def replace_line_starting_with(starting_with: str, replacement: str, string: str
     return re.sub(rf"^[ #]*{starting_with}.*$", replacement, string, count=1, flags=re.MULTILINE)
 
 
+# Downloading updated version of octopi.txt
+with urlopen('https://raw.githubusercontent.com/guysoft/OctoPi/devel/src/modules/octopi/filesystem/boot/octopi.txt') as file_response:
```

Comment:
Would be nice if you could move this string into a constant close to `octopi_config_file`, e.g. `OCTOPI_CONFIG_FILE_URL` or `OCTOPI_TXT_URL` or something :)